lib/core/kernel.nit

```diff
@@ -1045,12 +1045,13 @@ universal Char
 	#
 	#     assert 'A'.is_whitespace  == false
 	#     assert ','.is_whitespace  == false
-	#     assert ' '.is_whitespace  == true
+	#     assert ' '.is_whitespace  == true # space
+	#     assert ' '.is_whitespace  == true # non-breaking space
```

Comment:
Java does considers that 0xA0 is a whitespace, but Python does. I'm OK either ways

http://www.fileformat.info/info/unicode/char/00a0/index.htm
python -c 'print(u"\u00A0".isspace())'